active_model_serializers.gemspec

```diff
@@ -21,7 +21,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.1'
 
-  rails_versions = ['>= 4.1', '< 7.0']
+  rails_versions = ['>= 4.1', '< 7.1']
```

Comment:
might as well remove the max version.  only real benefit is it gets people to ask us to add a new version to CI

Reply:
@wasifhossain had a different [idea](https://github.com/rails-api/active_model_serializers/pull/2428#discussion_r770411827) but happy to go with the consensus.